spread_tests/integration/task.yaml

```diff
@@ -5,5 +5,6 @@ execute: |
   echo "Check that we are using the snapcraft snap"
   snapcraft_path="$(which snapcraft)"
   [ "$snapcraft_path" = "/snap/bin/snapcraft" ]
+  source /snapcraft/venv/bin/activate
```

Comment:
Oh, sorry, I merged before reading this question.
Yes, how would I install the requirements in the venv without activating it?